theme_test_custo/static/tests/tours/theme_menu_hierarchies.js

```diff
@@ -9,6 +9,7 @@ wTourUtils.registerEditionTour('theme_menu_hierarchies', {
     {
         content: 'Check Mega Menu is correctly created',
         trigger: 'iframe #top_menu a.o_mega_menu_toggle',
+        timeout: 20000,
```

Comment:
Oh indeed, when I first saw the runbot error I though that it was our automatic 30000 timeout of `registerEditionTour` which was still not good enough... but in fact it is only when you set `edition: true` :grimacing: Maybe this should have been done in `registerEditionTour` for when `edition: false` ? Increase the given first step's timeout ?

Anyways, let's r+ this one for now. We can still improve later and of course try to reduce the need for those timeout :grimacing: